### PR TITLE
update: Correct maxmemory setting and eviction policy

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

File: docs/products/redis/concepts/memory-usage.md [hunk 1]
```diff
@@ -25,29 +21,29 @@ high memory usage, and can result in a high change rate.
...
 that maximum is reached. All Aiven for Redis services have the eviction
 policy set to *No eviction* by default. This means that if you keep
 storing values, and never remove anything, the write operations will
 start failing when the maximum memory is reached.
 
 This is acceptable when data is consumed at a similar rate to how it is
-written. However, for other use cases `Evict all keys` with least
-recently used first (LRU) - which starts dropping old keys when
-`max memory` is reached - works better. Another way is to drop random
+written. However, for other use cases, using the allkeys-lru eviction policy - which
```
Comment (bogdanp05) [inline docs/products/redis/concepts/memory-usage.md:32]: `allkeys-lru`

**Review by bogdanp05 — CHANGES_REQUESTED** [review]
looks good, one nit

File: docs/products/redis/concepts/memory-usage.md (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/products/redis/concepts/memory-usage.md]: Done.

**Review by bogdanp05 — APPROVED** [review]
lgtm. Someone else needs to merge though, I don't have the rights

**Comment by harshini-rangaswamy** [comment]
> lgtm. Someone else needs to merge though, I don't have the rights

That is correct! Need a quick fellow tech writer review and approval to be able to merge.

**Review by ArthurFlag — APPROVED** [review]

**Pull request merged**